automation/_input/04_Powers_SAMPLE.yaml

```diff
@@ -1,31 +1,41 @@
 Template: # Power name        # Must be unique
-    Type: One of              # Passive, Vulny, Major, Minor, Adversary, Free, House. 
-    # Note: Type is required. For Companion actions, type indicates type req for dealer
+    Type: One of              # Required. Passive, Vulny, Major, Minor, Adversary, Free, House.
     Category: One or list     # One string or list in sub-cat order
+    Mechanic: Descrip         # Rule/effect. Save info added after. No ending periods.
+    Description: Descrip      # Flavor: 1-2 sentences
     XP: X
     PP: X                     # For multiple, implies 'OR'. Use [A, B] or `-`
     Prereq:                   # For multiple, implies 'OR'. Use [A, B] or `-`
         Role: X
-        Level: X              # Integer               
-        Skill: X > 1          #              
+        Level: X              # Integer
+        Skill: X > 1          #
         Power:                # Must match unique power name
-    To Hit: X                 # If not specified, assumed character's primary skill
-    Damage: X                 # Integer           
+    StatOverride:             # One of AGL, STR, Finesse, etc. followed by number
```

Comment:
Does Stat Override reduce the existing stat by the integer or does it auto-apply that number as a static? i.e. at char creation I take Clumsy -1 to my AGL so my AGL is -1, but then at next level I bump AGL by 1 to get my AGL to 0 - will the Clumsy Power continue to overwrite that change or does it reduce by this int?

Reply:
Current implementation replaces the value [here](https://github.com/CBroz1/TheGame/blob/main/automation/templates/bestiary.py#L250), but I can make edits to change to a sum